src/Microsoft.CodeAnalysis.Analyzers/CSharp/CSharpImmutableObjectMethodAnalyzer.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.CodeAnalysis.CSharp.Analyzers
 {
     using static CodeAnalysisDiagnosticsResources;
 
+    /// <summary>
+    /// RS1014: <inheritdoc cref="DoNotIgnoreReturnValueOnImmutableObjectMethodInvocationTitle"/>
```

Comment:
How does this help in Ctrl + , search? It does improve the quickinfo for the type names, which is valuable by itself, but wanted to know if there is some prefix I should add in Ctrl + , search for either the rule ID or the content/name of the title string to lead to results.

Reply:
You can do `Ctrl+,` and then type `CAXXXX` (`RS1014` in this case) and the file will appear in the list. I don't know the details of how the search is working. @sharwell Started this change on the codebase some time ago but it seems that it wasn't fully done or it got lost in refactorings/new rules addition. The rule explanation (the `<inheritdoc>` part) is not so helpful so we could remove it. With current way, if you call the type you see in the intellisense doc rule ID + rule title.